<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>previewPage</class>
 <widget class="QWidget" name="previewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="outputFilesList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="previewText"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
